website/demo-styles/container.scss

```diff
@@ -28,9 +28,11 @@
     line-height: 160px;
   }
 
-  & > .source > .el-container {
+  & > .source  > div > .el-container {
     margin-bottom: 40px;
-
+    &:last-child {
+      margin-bottom: 0;
+    }
```

Comment:
So why is the `> div` necessary?
You can't access the `.el-container` directly instead you need to locate it as the grandchild of `.source`?

Reply:
This interface display problem already exists in `element-ui`, because there is a layer of `div` node in HTML structure, so we can only find `el-container` through grandchildren.


Before change:
![image](https://user-images.githubusercontent.com/33242654/100338339-6ca1ff80-3013-11eb-875d-a7617993993d.png)

After change:
![image](https://user-images.githubusercontent.com/33242654/100338597-c0ace400-3013-11eb-9a54-a699afb68d53.png)